grails-app/domain/org/pih/warehouse/requisition/Requisition.groovy

```diff
@@ -113,6 +113,7 @@ class Requisition implements Comparable<Requisition>, Serializable {
     String monthRequested
 
     Integer replenishmentPeriod = 0
+    ReplenishmentType replenishmentType = ReplenishmentType.PUSH
```

Comment:
Change this to ReplenishmentTypeCode to be consistent with other enums. We eventually might add a static data class (ReplenishmentType) that should allow the user to use their own naming.

Reply:
@jmiranda Done